### DISCUSSION TITLE
Load limit for lists?

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#11123] [Q&A] [answered]
Can someone link me to the issue or relevant information about why we have a new limit for lists defaulted to 1000 caches?
I missed this topic completely and wondered about some maybe related error reports.

**Comment by fm-sys** [comment]
BTW, I just had a thought: why do users need to explicitly trigger load of more items? Would it be possible to just automatically load more results in the background if the user comes near the bottom like many other apps (e.g. Gmail) are doing it?

**Reply by Lineflyer** [reply]
The underlying problems are more than that, e.g. Changing the sorting of a list with 1000+ caches does not work. The caches currently not loaded will not be considered while re-sorting. Other reports tell, that something is fishy on history lists with >1000 entries (recent logs missing).

I only then was told by a user, that there is such a limit and "load more" button.

But the initial question remains: Why at all was this implemented? I could not find any issue while searching by now.

**Comment by eddiemuc** [comment]
This was implemented months ago to solve an old issue for load performance of really long lists. Don't remember issue id but I think it was filed.by magpiefourtytwo

Sorting should work nevertheless because criteria is included in sql query. If not, this must be a bug.

**Comment by eddiemuc** [comment, marked as answer]
Issue where load limit was discussed was #9015,  PR introducing the limit was #10147

**Comment by Lineflyer** [comment]
Will follow up in an issue.
